src/pages/ScholarshipForm.jsx

```diff
@@ -37,6 +39,8 @@ function ScholarshipForm() {
         max: 0,
       },
       website: '',
+      authId: user?.uid || null,
+      authEmail: user?.email || null,
```

Comment:
`auth` could be a little confusing. I'd prefer `author` or `owner`.

Reply:
Additionally, this violates the [scholarships data schema](https://github.com/beyondhb1079/s4us/blob/main/src/models/Scholarships.ts#L5) expected. I'm sure if you wrote a test that tries to save this, you won't be able to read `authId` and `authEmail` back using the `Scholarships` class.

The Scholarship model needs to be updated to include this. Also, considering that `firebase.auth.currentUser` may be null initially, these values would probably be better set in `onSubmit` instead and ensured that they're not empty.